packages/@okta/vuepress-plugin-my-okta/clientRootMixin.js

```diff
@@ -42,9 +46,10 @@ export default {
       });
       window.dispatchEvent(myOktaAccountFound)
 
-      const fences = document.querySelectorAll("[class*=language-]")
+      const fences = document.querySelectorAll("code")
       fences.forEach((fence, index) => {
         fences[index].innerHTML = fence.innerHTML.replace(/https:\/\/{yourOktaDomain}/gi, domain)
```

Comment:
Why not `fences[index].innerHTML = fence.innerHTML.replace(/https?:\/\/{yourOktaDomain}/gi, domain)` and skip doing it twice?

Reply:
Because `regex+bretterer=not smart`